src/Veldrid/GraphicsDevice.cs

```diff
@@ -967,6 +967,27 @@ public BackendInfoMetal GetMetalInfo()
 
             return info;
         }
+
+        /// <summary>
+        /// Updates CVDisplayLink active display using the display's origin and size.
+        /// </summary>
+        /// <param name="x"></param> x coordinate of the display's origin
+        /// <param name="y"></param> y coordinate of the display's origin
+        /// <param name="w"></param> display's width
+        /// <param name="h"></param> display's height
+        public virtual void UpdateActiveMonitor(int x, int y, int w, int h)
+        {
+            return;
+        }
+
+        /// <summary>
+        /// Calls CVDisplayLinkGetActualOutputVideoRefreshPeriod
+        /// </summary>
+        /// <returns>The actual refresh period of the currently selected CVDisplayLink display in seconds</returns>
+        public virtual double DisplayLinkGetActualOutputVideoRefreshPeriod()
+        {
+            return -1.0f;
```

Comment:
Why -1? Even `CVDisplayLinkGetActualOutputVideoRefreshPeriod` returns 0 if not running.

Reply:
The idea was that if `DisplayLinkGetActualOutputVideoRefreshPeriod` is called when not using MTL (which it shouldn't), a return value of -1 signifies some kind of error/something not applicable whereas 0 with DisplayLink means the display is not outputting. 
Am I overthinking it?